includes/polling/os/ciena-sds.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
Can you convert this file to the new format?  See LibreNMS/OS/Exa.php for an example.

Reply:
Yup. Done.